src/Arcus.Messaging.ServiceBus.Core/Extensions/ObjectExtensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Text;
+using Arcus.Messaging.Abstractions;
+using GuardNet;
+using Microsoft.Azure.ServiceBus;
+using Newtonsoft.Json;
+
+namespace Arcus.Messaging.ServiceBus.Core.Extensions
+{
+    public static class MessageExtensions
+    {
+        private const string JsonContentType = "application/json";
+
+        /// <summary>
+        ///     Creates an Azure Service Bus Message for a message body
+        /// </summary>
+        /// <param name="messageBody">Body of the Service Bus message to process</param>
+        /// <param name="operationId">Unique identifier that spans one operation end-to-end</param>
+        /// <param name="transactionId">Unique identifier that spans one or more operations and are considered a transaction/session</param>
+        /// <param name="encoding">Encoding to use during serialization. Defaults to UTF8</param>
+        /// <returns>Azure Service Bus Message</returns>
+        public static Message WrapInServiceBusMessage(this object messageBody, string operationId = null, string transactionId = null, Encoding encoding = null)
```

Comment:
Not sure how I feel about extensions on objects.

Reply:
It's good for making it discoverable imo, but when we have the service bus plugin then this should be handled for us